system/jlib/jthread.cpp

```diff
@@ -1841,7 +1840,7 @@ protected: friend class PipeWriterThread;
         if (et)
         {
             et->stop();
-            delete et;
```

Comment:
I am curious why it was coded this way.  I think it was to avoid errbufferthread->stop() from being called more than once, but I don't think that is going to cause any problems.